sonar-flex-plugin/src/test/java/org/sonar/plugins/flex/FlexSquidSensorTest.java

```diff
@@ -67,31 +69,30 @@ public void setUp() throws Exception {
     ActiveRules activeRules = new DefaultActiveRules(Collections.singletonList(ar));
     CheckFactory checkFactory = new CheckFactory(activeRules);
     FileLinesContextFactory fileLinesContextFactory = mock(FileLinesContextFactory.class);
-    when(fileLinesContextFactory.createFor(Mockito.any(InputFile.class))).thenReturn(mock(FileLinesContext.class));
+    FileLinesContext fileLinesContext = mock(FileLinesContext.class);
+    when(fileLinesContextFactory.createFor(Mockito.any(InputFile.class))).thenReturn(fileLinesContext);
     sensor = new FlexSquidSensor(checkFactory, fileLinesContextFactory);
     tester = SensorContextTester.create(TEST_DIR);
     logTester.clear();
   }
 
   @Test
-  public void analyse() throws FileNotFoundException {
+  public void analyse() throws IOException {
     DefaultFileSystem fs = new DefaultFileSystem(TEST_DIR);
+    fs.setEncoding(Charset.defaultCharset());
```

Comment:
That makes the test platform-dependent: they may succeed or fail depending on the platform. A better choice is to use a given charset, e.g. `StandardCharsets.UTF-8`.

Reply:
Good point!